q_yaar_platform/jwt_auth/services/helper.py

```diff
@@ -0,0 +1,342 @@
+from django.conf import settings
+import jwt
+import logging
+import uuid
+
+from django.core.exceptions import ValidationError
+from django.core.validators import validate_email
+
+from account.models import PlatformUser
+from account.services.interfacer import (
+    svc_account_check_if_user_with_email_exists,
+    svc_account_create_platform_user,
+    svc_account_get_platform_user_by_email,
+    svc_account_get_platform_user_by_id,
+    svc_account_get_serialized_platform_user,
+)
+from common.constants import UserRolesType
+from common.phonenumber import is_valid_indian_number
+from jwt_auth.authentication import JWTRefreshToken
+from profile_player.models import PlayerProfile
+from profile_player.services.interfacer import (
+    svc_player_create_player_for_platform_user,
+    svc_player_get_player_for_platform_user,
+    svc_player_get_serialized_player_profile,
+    svc_player_update_player,
+)
+
+from .error_codes import ErrorCode
+
+
+logger = logging.getLogger(__name__)
+
+
+######################################################################################################################
+##########################################   PRIVATE FUNCTIONS   #####################################################
+######################################################################################################################
+
```

Comment:
Better to segregate private and public helper functions for best code reusability.